libs/tree-sitter-wing/grammar.js

```diff
@@ -186,7 +187,7 @@ module.exports = grammar({
 
     resource_definition: ($) =>
```

Comment:
Followup to remove `resource_definition` to consolidate `class_definition`?

Reply:
But of course... but not in this PR